tests/integration/500.command.dev.test.js

```diff
@@ -1,11 +1,14 @@
 // Handlers are meant to be async outside tests
 /* eslint-disable require-await */
+const { promises: fs } = require('fs')
```

Comment:
NIT: 

```suggestion
const fs = require('fs/promises')
```

Reply:
I don't think that syntax is supported in Node 12

Reply:
Confirmed. fs/promises was added in Node 14